py-polars/polars/selectors.py

```diff
@@ -155,7 +155,7 @@ def _expand_selectors(frame: DataFrame | LazyFrame, *items: Any) -> list[Any]:
         if is_selector(item):
             selector_cols = expand_selector(frame, item)
             expanded.extend(selector_cols)
-        else:
+        elif item is not None:
```

Comment:
@alexander-beedie this PR touches on your selects here. I don't think it has any unintended consequences. The prior behavior was for `_expand_selectors(None)` to return `[None]`, which is undesirable when you're calling it on a keyword arugment that defaults to `None` (as is the case in this PR). This change now excludes `None`s from the expansion. I don't think there is any instance in which `None` is an acceptable column anyway.

Reply:
Adding this drive-by is a bad idea. You handled it correctly in your function (check for None). Improving the selector expansion can be done in a separate PR - it is not required here.